panel/src/panel/content.js

```diff
@@ -1,25 +1,23 @@
 import { length } from "@/helpers/object";
-import { reactive, watch } from "vue";
+import { reactive } from "vue";
+import throttle from "@/helpers/throttle.js";
 
 /**
  * @since 5.0.0
  */
 export default (panel) => {
 	const content = reactive({
 		/**
-		 * API endpoint to handle content changes
-		 */
-		get api() {
-			return panel.view.props.api;
-		},
-
-		/**
-		 * Returns all fields and their values that
-		 * have been changed but not yet saved
-		 *
+		 * Returns an object with all changed fields
+		 * @param {String} api
 		 * @returns {Object}
 		 */
-		get changes() {
+		changes(api = panel.view.props.api) {
```

Comment:
`Question:` Not sure about this but I somehow remember that the default will not be applied when the method is called but when the module code is parsed the first time. Do you know if I am wrong or right with this?

Reply:
I'm not sure I understand what you mean.

Reply:
Whether the default of `api` will be `panel.view.props.api` from the moment when the method gets called or whatever `panel.view.props.api` contains at the moment the content module gets initialized for the first time.

Reply:
Oh, now I get it. That's interesting. That would mean that we should rather leave it undefined and then set the default in each method call, right? Right now, it's not super important, because the view will set the api endpoint explicitely on all calls, but in other places it might lead to unwanted problems indeed.

Reply:
I've moved the default api endpoint assignment into the method calls.